NEWS

```diff
@@ -1,8 +1,13 @@
-Unreleased
-==========
+0.34.0
+======
 
-* Dropped support for Python 3.7 and earlier.
+* Dropped support for Python 3.6 and earlier.
```

Comment:
As we made revert I prefer to not link the related pull requests to avoid confusion.